apps/svelte.dev/src/routes/examples/[...path]/+page.server.ts

```diff
@@ -1,5 +1,7 @@
 import { redirect } from '@sveltejs/kit';
 
+export const prerender = false;
```

Comment:
ah yea, that was missing, of course!

I'm wondering though if we should move it into the `handle` hook aswell and delete this file, because AFAIK all redirects happen inside `handle` right now.

Reply:
Thanks for the suggestion. For consistency, it does seem better to consolidate the redirects into the `handle` hook.  I’ll update the PR to move the redirect logic there and remove the file accordingly.

Reply:
I’ve completed the changes. Please review at your convenience.